packages/amplify-cli/src/commands/push.ts

```diff
@@ -11,7 +11,7 @@ import { updateCognitoTrackedFiles } from '../extensions/amplify-helpers/update-
 /**
  * Download and unzip deployment bucket contents to #current-cloud-backend so amplify status shows correct state
  */
-const syncCurrentCloudBackend = async (context: $TSContext): Promise<void> => {
+export const syncCurrentCloudBackend = async (context: $TSContext): Promise<void> => {
```

Comment:
I know this is being exported for assertions in the unit tests, but exporting functions that are expected to be 'internal' can cause problems in the future. Is there any way we can skip exporting this and updateTrackedFiles (e.g. you can mock `updateCognitoTrackedFiles` instead)

Reply:
I did update the updateTrackedFiles, for syncCurrentCloudBackend I prefer to not test on inner methods calls as that might change and miss the point of the test, I did however moved in a separate module